bitnami/airflow/templates/config/configmap.yaml

```diff
@@ -14,7 +14,7 @@ metadata:
 data:
 {{- if .Values.config }}
   airflow.cfg: |-
-    {{ .Values.config | indent 4 }}
+{{ .Values.config | indent 4 }}
```

Comment:
Please change it to use `nindent` instead of `indent`

Reply:
hi, did you mean to change it into?
```
  airflow.cfg: |- {{ .Values.config | indent 4 }}
```
If yes, i don't think it is more readable.

Reply:
I added a new comment about this below